webapp/channels/src/components/sidebar/sidebar_list/sidebar_list.tsx

```diff
@@ -318,7 +319,7 @@ export default class SidebarList extends React.PureComponent<Props, State> {
     };
 
     navigateChannelShortcut = (e: KeyboardEvent) => {
-        if (e.altKey && !e.shiftKey && !e.ctrlKey && !e.metaKey && (Keyboard.isKeyPressed(e, Constants.KeyCodes.UP) || Keyboard.isKeyPressed(e, Constants.KeyCodes.DOWN))) {
+        if (e.altKey && !e.shiftKey && !e.ctrlKey && !e.metaKey && (isKeyPressed(e, Constants.KeyCodes.UP) || isKeyPressed(e, Constants.KeyCodes.DOWN))) {
```

Comment:
Are these changes something to do with tree shaking or are these simply stylistic?

Reply:
This is for the tree shaking

Reply:
I don't think that will actually matter since it still has to import and run all of `utils/keyboard` regardless of if we actually use it. You'd have to split it into multiple files to not load everything because of how dynamic JS is.

Similarly, we used to have the ESLint plugin which changed `import {isEmpty} from 'lodash'` to `import isEmpty from 'lodash/isEmpty'` because the first version can't be tree shaken but the second one can be

Reply:
Oh the above named import is actually tree shake able ref : https://web.dev/articles/reduce-javascript-payloads-with-tree-shaking#importing_only_whats_needed

Reply:
Thanks for the link! That's really interesting since I didn't realize it was possible to do that inside of a file.

I think we might be missing a critical part for that though. We don't have `modules: false` in the configuration for `@babel/preset-env` which I think is required for that. There's also a note in that document though about us needing to potentially set the `sideEffects` option for it though, and I don't know if that's something that we need, so we should probably look into enabling that separately